recipe/patches/0001-Add-conda-forge-version.patch

```diff
@@ -0,0 +1,25 @@
+# HG changeset patch
+# Date 1627369254 -10800
+#      Tue Jul 27 10:00:54 2021 +0300
+# Branch conda
+# Node ID 3bd4e146c802b6a2549bedb139881814845cbb23
+# Parent  a62de68281e3bc881cddc1d56646567dc543e3cd
+add 'packaged by conda-forge' like in CPython feedstock
+
+diff -r a62de68281e3 -r 3bd4e146c802 pypy/module/sys/version.py
+--- a/pypy/module/sys/version.py	Fri Jul 02 18:18:41 2021 +0300
++++ b/pypy/module/sys/version.py	Tue Jul 27 10:00:54 2021 +0300
+@@ -55,11 +55,11 @@
+     ver = "%d.%d.%d" % (PYPY_VERSION[0], PYPY_VERSION[1], PYPY_VERSION[2])
+     if PYPY_VERSION[3] != "final":
+         ver = ver + "-%s%d" %(PYPY_VERSION[3], PYPY_VERSION[4])
+-    template = "%d.%d.%d (%s, %s, %s)\n[PyPy %s with %%s]" % (
++    template = "%d.%d.%d | packaged by conda-forge | (%s, %s, %s)\n[PyPy %s with %%s]" % (
```

Comment:
I think this broke package builds, see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=357434&view=logs&jobId=856e153d-59ad-5597-92a7-c22c7f7f2082&j=856e153d-59ad-5597-92a7-c22c7f7f2082&t=2f20f7ee-86c8-5a1e-38c4-b2540d093bc9
`ValueError: failed to parse PyPy sys.version: '3.7.10 | packaged by conda-forge | (77787b8f, Jul 28 2021, 19:20:52)\n[PyPy 7.3.5 with GCC 9.3.0]'`

Reply:
Oy oy oy. Sorry.

Reply:
think it'll need to match this regex https://github.com/python/cpython/blob/3.9/Lib/platform.py#L995-L998 ?